### DISCUSSION TITLE
Using TableReader + Macros to read an entire directory of CSVs?

### DISCUSSION THREAD

**Discussion by Firelord710** [squidfunk/mkdocs-material#7471] [Q&A] [answered]
Hey guys.

I have a directory filled with CSVs that are created on website compilation (using MKDocs Material + Cloudflare Pages).

I am struggling to figure out how to pass through the entire directory of CSVs to table reader. 

My first thought was creating key value pairs that the value would be the dir for the csv, but this was cut short by not being able to figure out how to create a key value dictionary from two Pandas Dataframes containing the keys and values separately. 

I'd appreciate any help or direction! 

Thanks again.

**Comment by squidfunk** [comment, marked as answer]
Closing due to [double posting](https://github.com/mkdocs/mkdocs/discussions/3816), as [per our policy](https://squidfunk.github.io/mkdocs-material/contributing/#incomplete-issues-and-duplicates). Please don't do that.
